db/migrate/20200521082853_rename_cent_os7_install_media.rb

```diff
@@ -0,0 +1,9 @@
+class RenameCentOs7InstallMedia < ActiveRecord::Migration[6.0]
+  def up
+    Medium.find_by(name: "CentOS mirror").try(:update_attribute, :name, "CentOS mirror (7.x)")
```

Comment:
You can use 
```suggestion
    Medium.where(name: "CentOS mirror").update_all(name: "CentOS mirror (7.x)")
```
which will work even if there are no matches (and also update the db without triggering callbacks)